features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96_CellularStack.cpp

```diff
@@ -401,3 +410,21 @@ nsapi_error_t QUECTEL_BG96_CellularStack::gethostbyname_async_cancel(int id)
     return NSAPI_ERROR_OK;
 }
 #endif
+
+void QUECTEL_BG96_CellularStack::ip2dot(const SocketAddress &ip, char *dot)
```

Comment:
How about moving this metod into CellularUtil?

Reply:
Typically you'd use `get_ip_address` or `ip6tos`. For now, this is assumed to be BG96 specific implementation and can be moved later if needed.